### ISSUE TITLE
More Uses For the Badmin Stones

### ISSUE THREAD

**Issue by carlopault** [HippieStation/HippieStationdeprecated2020#12617]
The Badmin Stones are interesting and don't see much use outside of Thanos gamemode. Perhaps they can be used as part of some new artifacts, i.e. Loki's Scepter. It could allow the user to mindslave anyone via targeting their chest and waiting 1 second, shoot blue fireballs, and summon armor on the user. Clicking the scepter with a crowbar will drop whatever we have as the equivalent to the mind stone. I don't have any ideas as to how the staff could be obtained outside of just slapping it in the spellbook.

**Comment by AnthonyJNgo** [comment]
How would they be found or implemented? Please don't say mining or science

**Comment by MrEpicSax** [comment]
> 
> 
> How would they be found or implemented? Please don't say mining or science

Space ruins?

**Comment by carlopault** [comment]
> How would they be found or implemented? Please don't say mining or science

Supermatter Stone: Superheating the supermatter crystal to *insert very hot temperature, idk how easy it is to get specific temperatures*. When it super-maxcap explodes, instead of leaving behind a singularity like we have it do right now, it leaves behind the Supermatter Stone. 

Lag Stone: Spawned at a random location once the server reaches quadruple the time dilation that it was at the start of the round.  

Syndicate Stone: Slap it in the uplink for 20 TC and an announcement stating that "99#&! Joules of SYNDICATE tainted energy detected! X has purchased the Syndicate Stone at the X!". Only 1 Syndicate Stone can be purchased. Once a traitor purchases it then other traitors can't purchase it. Think of some cool theme to play during the announcement, like how the Nanosuit has an epic theme. 

Ghost Stone: If ~25 ghosts orbit a single mob then a series of announcements are made warning the crew that the power of the dead is growing too high. The Ghost Stone will spawn on the mob and unleash a mega stun shockwave that pushes objects. 5 randomly selected ghosts that are orbiting the mob are made into randomly selected SIMPLEMOB/hostile mobs. If hostile mobs are too powerful then select less powerful shit to spawn em as. 

Clown Stone: If the Clown ever somehow manages to wield the Staff of the Honkmother and Pie of Destruction at the same time, the Clown slowly transforms into a godlike being that is imbued with the powers of the Clown Stone and drops it when killed. 

Bluespace Stone: This cannot not relate to Science. If the Bluespace Artillery is constructed and fired into a bluespace portal, a shockwave shakes the entire station and teleports 50% of players to random locations. The Bluespace Stone spawns at the site of the portal and protective bluespace portals (3x3 around the stone) are spawned around it that lead to random locations across all Z-levels (except locations with super OP shit like the CentComm Armory or something). The portals can be deactivated by throwing a bluespace crystal at them. This is all coupled with an announcement stating "Massive bluespace energies detected at Space Station 13! What the hell did you retards do!?"

None of the stones will spawn through these ways if the wizard has bought the Badmin Gauntlet.

**Comment by Mastergodown** [comment]
No. Syndicate Stone, while that IS clever, shouldnt be an effortless uplink item.

**Comment by carlopault** [comment]
> No. Syndicate Stone, while that IS clever, shouldnt be an effortless uplink item.

Got any ideas?

**Comment by Mastergodown** [comment]
Perhaps if the syndie comms agent base's nuke is set off while there is 20 TC in the room it will create it, but maybe something different. I also like the lag stone requirements, but I dont think it should encourage players to lag the server to get it, but I dont have any other thoughts.
